workflow.rst

```diff
@@ -265,6 +265,36 @@ registry in the constructor::
         }
     }
 
+Or use the typed + named parameter injection::
```

Comment:
I would put this paragraph before the one about the registry.

Reply:
:heavy_check_mark: